ecosystem/attestation-station/using-sdk/index.mjs

```diff
@@ -0,0 +1,118 @@
+#! /usr/local/bin/node
+
+// wagmi prefers to use JavaScript modules, see https://wagmi.sh/react/module-types
+
+// Read .env
+const dotenv = await import("dotenv")
```

Comment:
```suggestion
import dotenv from "dotenv"
```
Just a nit doing it this way is more idiomatic

Reply:
I'll change it for packages that support it (which, BTW, atst does not).

```
ori@Oris-MacBook-Pro using-sdk % node index.mjs
file:///Users/ori/data/mergers/optimism-tutorial/ecosystem/attestation-station/using-sdk/index.mjs:13
import atst from "@eth-optimism/atst"
       ^^^^
SyntaxError: The requested module '@eth-optimism/atst' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)

Node.js v18.14.2
```

I know I just import what I need, but I want to keep the package identifier. It makes the code clearer when you're a clueless noob, which is my target audience.